src/project/csprojReader.ts

```diff
@@ -74,6 +74,26 @@ export default class CsprojReader extends ProjectReader {
         return Number(versionMatch[0]) >= 6;
     }
 
+    /**
+     * Whether the 'ImplicitUsings' option is set to `enable`
+     *
+     * @returns If the 'ImplicitUsings' option is set to `enable`
+     */
+    public async useImplicitUsings(): Promise<boolean> {
+        try {
```

Comment:
nitpick:
The only risky operation seems to be `this.getPropertyGroups()` but the `try` wraps some more lines.
Declaring `propertyGroupWithImplicitUsings` (which could be tedious) outside the try/catch block will allow to move a couple of lines outside. The gain is in term of readability.
```typescript
let propertyGroupWithImplicitUsings : PropertyGroup;
try {
    const propertyGroups = await this.getPropertyGroups();
    const propertyGroupWithImplicitUsings = propertyGroups?.find(p => p.ImplicitUsings);
} catch (errParsingXml) {
    console.error('Error parsing project xml', errParsingXml);
    return false;
}

if (!propertyGroupWithImplicitUsings?.ImplicitUsings) return false;

return propertyGroupWithImplicitUsings.ImplicitUsings[0] === 'enable';
```

Reply:
These are 2 different ways of working with try-catch blocks.
Yours makes catching more explicit, but requires variable declaration outside of the try-block, which means one has to think ahead:

```ts
let propertyGroups: PropertyGroup[] | undefined;
let propertyGroupWithImplicitUsings: PropertyGroup | undefined;

try {
    propertyGroups = await this.getPropertyGroups();
    propertyGroupWithImplicitUsings = propertyGroups?.find(p => p.ImplicitUsings);
} catch (errParsingXml) {
    console.error('Error parsing project xml', errParsingXml);
    return false;
}

if (!propertyGroupWithImplicitUsings?.ImplicitUsings) return false;

return propertyGroupWithImplicitUsings.ImplicitUsings[0] === 'enable';
```

Yours is the more readable one, but requires a bit more work. It would be nice to have a auto-refactor step for this.
I'll change it later today, or sometime tomorrow.